engine/script/src/script_sys.cpp

```diff
@@ -541,13 +542,26 @@ union SaveLoadBuffer
      * end
      * ```
      */
-    int Sys_GetSysInfo(lua_State* L)
+    static int Sys_GetSysInfo(lua_State* L)
     {
         int top = lua_gettop(L);
 
         dmSys::SystemInfo info;
         dmSys::GetSystemInfo(&info);
 
+        bool ignore_secure_values = false;
+
+        if ( top >= 1)
+        {
+            luaL_checktype(L, 1, LUA_TBOOLEAN);
+            ignore_secure_values = lua_toboolean(L, 1);
+        }
+
+        if (!ignore_secure_values)
+        {
+            dmSys::GetSecureInfo(&info);
+        } 
+
```

Comment:
Keep behaviour by default as it was and make getting for the secure values optional.